assets/airports/lszh.json

```diff
@@ -0,0 +1,1423 @@
+                ["N47.475667", "E8.536000", "1390ft"],
+                ["N47.449333", "E8.554167", "1385ft"]
+            ],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "ALBIX": {
+            "icao": "ALBIX",
+            "name": "Albix",
+            "suffix": {"10":"1C"},
+            "rwy": {
+                "10": ["^_RWY10KLO02.1DME", "_KLO085005", ["^ZH502", "A40+|S210-"], "_TRA167024", "_(TRA167024)049005"]
+            },
+            "body": [],
+            "exitPoints": {
+                "ALBIX": [["ALBIX", "A100+"]]
+            },
+            "draw": [
+                ["ZH502", "_(TRA167024)049011.5", "ALBIX*"]
+            ]
+        },
+        "DEGES": {
+            "icao": "DEGES",
+            "name": "Deges",
+            "suffix": {"10":"1D", "16":"2S", "28":"2W", "32":"3L", "34":"3F"},
+            "rwy": {
+                "10": ["^_RWY10KLO02.1DME", "_KLO085005", ["^ZH502", "A40+"]],
+                "16": ["^_RWY16KLO01DME", "_KLO085005", ["^ZH502", "A40+"]] ,
+                "28": [["^_RWY28KLO03DME", "S210-"], ["_KLO253005", "S210-"], ["^ZH552", "S210-"], "_KLO232005", "^KLO", ["MOMOL","A80+"]],
+                "32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["^KLO", "A50+"], ["MOMOL", "A80+"]],
+                "34": [["^_RWY34KLO04DME", "A35+|S210-"], "_TRA190011", ["^_TRA190012.5", "A45+"], "_KLO269005.5", ["^KLO", "A50+"], ["MOMOL", "A80+"]]
+            },
+            "body": ["KOLUL", ["ZH504", "A50+"], ["^ZH525", "A70+"]],
+            "exitPoints": {
+                "DEGES": [["DEGES", "A80+"]]
+            },
+            "draw": [
+                ["_KLO085005", "ZH502", "KOLUL","ZH504","ZH525","DEGES*"]
+            ]
+        },
+        "VEBIT": {
+            "icao": "VEBIT",
+            "name": "Vebit",
+            "suffix": {"10":"2E", "16":"3S", "28":"3W", "32":"2N", "34":"2H"},
+            "rwy": {
+                "10": ["^_RWY10KLO02.1DME", "_WIL053030", ["^BREGO", "A50+"]],
+                "16": [["^_RWY16KLO01DME", "S210-"], ["_KLO360001.5", "S210-|A40+"], "_WIL053027", ["^BREGO", "A50+"]],
+                "28": ["^_RWY28KLO03DME", "_BREGO053004", "^BREGO"],
+                "32": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["^BREGO", "A50+"]],
+                "34": ["^_RWY34KLO02DME", ["^_RWY34KLO04DME", "A35+"], "_TRA190011", ["^BREGO", "A50+"]]
+            },
+            "body": [["ZH554", "A60+"], ["ZH558", "A70+"]],
+            "exitPoints": {
+                "VEBIT": ["VEBIT"]
+            },
+            "draw": [
+                ["BREGO", "ZH554", "ZH558", "VEBIT*"]
+            ]
+        },
+        "WIL": {
+            "icao": "WIL",
+            "name": "Willisau",
+            "suffix": {"10":"2D", "16":"3R", "28":"3V", "32":"2M", "34":"2G"},
+            "rwy": {
+                "10": ["^_RWY10KLO02.1DME", "_WIL053030", ["^BREGO", "A50+"]],

```

Comment:
- We need a restriction to cross above 4000ft and max 210kts
- `BREGO` is not a fly-over fix